### PR TITLE
Check the pagination of product in the Front Office

### PR DESCRIPTION
**Pull request by hadrich-hatem**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.3.x
| Description?  | This PR allows to : Edit the number of product per page. Disable the friendly URL in the Back Office. Check the pagination of product in the Front Office.
| Type?         | improvement
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-4847
| How to test?  | Run this script path=full/02_product/10_check_pagination_of_product_in_FO.js npm run specific-test -- --URL=BackOfficeURL

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8881)

### CODE REVIEW

File: tests/E2E/test/campaigns/common_scenarios/product.js [hunk 1]
```diff
@@ -157,5 +157,16 @@ module.exports = {
           .then(() => client.checkSortProduct())
       });
     }, 'product/product');
+  },
+
+  checkPaginationFO(client, productPage, buttonName, pageNumber) {
+    let selectorButton = buttonName === 'Next'? productPage.pagination_next : productPage.pagination_previous;
```
Comment (fatmaBouchekoua) [inline tests/E2E/test/campaigns/common_scenarios/product.js]: A little space after 'Next'

**Review by fatmaBouchekoua — CHANGES_REQUESTED** [review]

File: tests/E2E/test/campaigns/common_scenarios/product.js (hunk 1 above)
Comment (hadrich-hatem) [reply tests/E2E/test/campaigns/common_scenarios/product.js]: I added a space after 'Next'

**Review by fatmaBouchekoua — DISMISSED** [review]

File: tests/E2E/test/campaigns/common_scenarios/product.js [hunk 2]
```diff
@@ -157,5 +157,16 @@ module.exports = {
...
       });
     }, 'product/product');
+  },
+
+  checkPaginationFO(client, productPage, buttonName, pageNumber) {
+    let selectorButton = buttonName === 'Next' ? productPage.pagination_next : productPage.pagination_previous;
+    test('should click on "' + buttonName + '" button', () => {
+      return promise
+        .then(() => client.isVisible(selectorButton))
+        .then(() => client.clickPageNextOrPrevious(selectorButton))
```
Comment (Quetzacoalt91) [inline tests/E2E/test/campaigns/common_scenarios/product.js]: Reported by Codacy, a `;` is missing here.

**Comment by hadrich-hatem** [comment]
Hello @Quetzacoalt91,
I have added the missing semicolon.
Thanks for your review.

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thanks @hadrich-hatem !

**Pull request merged**